README.md

```diff
@@ -7,6 +7,7 @@ Provides base OS, patches and stable nginx for quick and easy spinup.
 - Ubuntu used by default
 - Alpine builds available tagged as `-alpine`
 - Centos builds available tagged as `-centos`
+- Ubuntu with ngx_http_js_module tagged as `-njs`
```

Comment:
@kmashint : I noted this in the JIRA as well but the name needs to change to be more consistent with the rest. I will get back to you tomorrow.